internal/cmd/olmv1.go

```diff
@@ -31,11 +31,21 @@ func newOlmV1Cmd(cfg *action.Configuration) *cobra.Command {
 	}
 	createCmd.AddCommand(olmv1.NewCatalogCreateCmd(cfg))
 
+	updateCmd := &cobra.Command{
+		Use:   "update",
```

Comment:
alternatively we could use `set` to be more inline with `kubectl` behavior

Reply:
This is kinda philosophical about what the CLI is trying to do. Is it just trying to send simple changes to a resource in kubernetes? Or is it trying to orchestrate an OLMv1-specific workflow?

In my head, it is the latter. But the hope is that implementing the latter is mostly: (1) make simple change, (2) tell user about progress of that change, (3) tell user whether it worked or not.

If we're shooting for the workflow style naming, my suggestion would be "upgrade" for extensions, but there isn't really an upgrade workflow for catalogs, so this gets into the whole bikeshed about the structure of the CLI subcommand tree.

Reply:
I think stick with `update` for now and we can decide on the structure later.

Reply:
Is this comment sorted out?
Can we close it to make easier for others follow up?
What should I look on it?

Reply:
> Is this comment sorted out? What should I look at? WDYT about closing this conversion to make it easier to follow up the PR?

When you say it would be easier to follow up the PR what do you mean exactly? I don't want to start a new long thread here about how people approach reviewing PRs, but I'm curious.

If I start reviewing a PR I would look at both resolved and unresolved previous comments and If I saw situation like in this thread, I'd assume author has agreed with moving on with `update`. If I had any additional valuable input about that, I'd still add a comment here, so I'm not sure what the difference is.